group.yml

```diff
@@ -136,6 +136,15 @@ repos:
     content_set:
       default: rhel-server-rhscl-7-rpms
       ppc64le: rhel-7-server-for-power-le-rhscl-rpms
+  rhel-openstack-rpms:
+    conf:
+      baseurl:
+        ppc64le: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/power-le/7/7Server/ppc64le/openstack/14/os/
+        x86_64: http://pulp.dist.prod.ext.phx2.redhat.com/content/dist/rhel/server/7/7Server/x86_64/openstack/14/os/
+      enabled: 1
```

Comment:
@adammhaile am I right in remembering you saying this would enable the repo for *all* images? Don't think we need this, right?